tkn/tkn.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: tkn
+spec:
+  inputs:
+    params:
+      - name: tkn-image
+        description: tkn CLI container image to run this task
+        default: gcr.io/tekton-releases/dogfooding/tkn
+      - name: ARGS
+        type: array
+        description: tkn CLI arguments to run
+        default: ["help"]
```

Comment:
Should we have no `default` for the task, thus making the `ARGS` a required argument.

Reply:
Removed.